docs/_sidebar.md

```diff
@@ -1,6 +1,7 @@
 <!-- docs/_sidebar.md -->
 * [Home](/)
 * [Install](install.md)
+* [Contributing](contributing.md)
```

Comment:
Can we make `Contributing` the last page below the whole `Configuration` section.
Can we also move the `Logging` section out of the `Configuration` section and above `Contributing`.

Reply:
I moved the Contributing section. I don't think we should move the logging section out of configuration because we still need to "configure" the `MANZAN_DEBUG_LEVEL` variable